quickstarts/apache-flink/config.yml

```diff
@@ -0,0 +1,40 @@
+id: 
```

Comment:
We generate the ID once the PR is merged - you can go ahead and delete this line!

Reply:
id deleted from the config file.